wgpu/src/macros.rs

```diff
@@ -78,7 +78,14 @@ macro_rules! include_spirv_raw {
     };
 }
 
-/// Macro to load a WGSL module statically.
+/// Load WGSL source code from a file at compile time, the loaded path is relative to the source
+/// path.
```

Comment:
I think it'd be useful to mention the `std` behavior that this depends on, since that should have the most precise explanation. Also, "the source path" is a bit ambiguous if you don't already know what it means.

```suggestion
/// Load WGSL source code from a file at compile time.

/// The loaded path is relative to the path of the file containing the macro call,
/// in the same way as [`include_str!`] operates.
```

Reply:
updated :)